src/ddscxx/include/dds/core/policy/CorePolicy.hpp

```diff
@@ -63,14 +63,6 @@ class policy_id { };
 template <typename Policy>
 class policy_name { };
 
-
-
-#ifdef OMG_DDS_EXTENSIBLE_AND_DYNAMIC_TOPIC_TYPE_SUPPORT
```

Comment:
I removed this because I wanted the new functionality to compile without having to pull in the typeconsistency stuff which was not ready yet.
As you explained this to me, and I have re-added these back in.